include/pybind11/detail/native_enum_data.h

```diff
@@ -36,7 +36,7 @@ class native_enum_data {
     std::string was_not_added_error_message() const {
         return "`native_enum` was not added to any module."
                " Use e.g. `m += native_enum<...>(\""
-               + enum_name_encoded + "\")` to fix.";
+               + enum_name_encoded + "\", py::native_enum_kind::IntEnum)` to fix.";
```

Comment:
This might give an impression that the `IntEnum` kind is preferred, which isn't really true, right? Should this just use a placeholder ellipsis , or full suggestion of both versions?  (e.g., "Use `m += native_enum<...>("Name", Enum)` or `m += native_enum<...>("Name", IntEnum)` to fix)

Reply:
Changed to `...`. — I wasn't sure about this myself before. Considering that the `+=` part is the critical part of the error message, I believe using `...` is best (will distract people the least from the critical part).